Library/Homebrew/shims/linux/super/cc

```diff
@@ -261,8 +262,12 @@ class Cmd
     args = path_flags("-L", library_paths)
     case mode
     when :ld
+      args << "--dynamic-linker=#{prefix}/lib/ld.so"
+      args << "-rpath" << rpath_paths
       args << "-no_weak_imports" if no_weak_imports?
     when :ccld, :cxxld
+      args << "-Wl,--dynamic-linker=#{prefix}/lib/ld.so"
+      args << "-Wl,-rpath," + rpath_paths
```

Comment:
Yes. It shouldn't happen I think, but better to be paranoid. Thanks for catching that case.